agcaballero/portfolio/src/main/webapp/script.js

```diff
@@ -72,6 +72,129 @@ function testRandomPost(trials = 10000) {
 
 testRandomPost(10000);  // runs 10000 trials on randomPost()
 
+/** Creates a Map using Map API */
+function createMap() {
+  const europeLatLng =
+      new google.maps.LatLng({lat: 48.499998, lng: 23.3833318});
+  // the locations of the photos
```

Comment:
Maybe comment on why these are not stored in a database? For example:

```
These are hardcoded due to time constraints. Long term, these should be stored in a database.
```

Reply:
Thank you. I will add that. Currently, it's not stored in a database just because I was going to work on the server-side aspect of it today. However, I will add that comment as a place-holder.

Reply:
Additionally, I was wondering if it would be better to store these in a CSV or in datastore.

Reply:
Done